fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -365,9 +365,7 @@ private void refreshCardinality() {
             Set<ApiFilter> filters,
             PaginationParameters paginationParameters
     ) {
-        int perPage = paginationParameters.getPerPage();
-        validatePerPage(perPage);
```

Comment:
We could put validation in places where `perPage` is used, in this case `getResultsPage`, couldn't we?